pom.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.wso2</groupId>
+        <artifactId>wso2</artifactId>
+        <version>5</version>
+    </parent>
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.carbon.utils</groupId>
+    <artifactId>carbon-utils-parent</artifactId>
+    <version>5.2.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+    <name>WSO2 Carbon Kernel - Utils Parent</name>
+    <url>http://wso2.com</url>
+
+    <scm>
+        <url>https://github.com/wso2/carbon-utils.git</url>
+        <developerConnection>scm:git:https://github.com/wso2/carbon-utils.git</developerConnection>
+        <connection>scm:git:https://github.com/wso2/carbon-utils.git</connection>
+        <tag>HEAD</tag>
+    </scm>
+
+    <modules>
+        <module>components/org.wso2.carbon.utils</module>
```

Comment:
IMHO we don't need a separate feature for this. We can pack this either from config-resolver or kernel feature